docs-src/cloud/namespace-setup-considerations.md

```diff
@@ -17,7 +17,7 @@ Customers should consider not only a Namespace naming convention but also how to
 Before considering an appropriate Namespace configuration, you should be aware of the following constraints:
 
 - Each Temporal account has a default limit of 10 Namespaces.
-  You can request an increase by [creating a ticket for Temporal Support](/cloud/introduction/support#support-ticket).
+  You can request an increase by [creating a ticket for Temporal Support](/cloud/support#support-ticket).
```

Comment:
this is a docs-src node - link to the id of the node - NOT the final build slug.